mis_builder_budget/models/mis_report_instance.py

```diff
@@ -118,4 +118,9 @@ def drilldown(self, arg):
                     "view_mode": "list",
                     "target": "current",
                 }
+            elif period.source == SRC_MIS_BUDGET_BY_ACCOUNT:
+                arg["context"] = {
+                    "default_budget_id": period.source_mis_budget_by_account_id.id,
+                    "default_account_id": arg.get("account_id"),
+                }
```

Comment:
Mutating `arg` here does not sound right. Could this be propagated with `with_context` ?

Reply:
Done.